peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -77,6 +77,17 @@ <h1 class="py-4">{{ document.title }}</h1>
                         {% endif %}
                       {% endblock %}
 
+                      {% block document-metadata-content-alternative-names %}
+                        <dt>{% trans 'Alternative names' %}</dt>
+                        <dd>
+                          {% for alternative_name in document.alternative_names.all %}
```

Comment:
I think each should be in their own div, so they're on a separate line, since it may not be clear where one ends and other starts otherwise.